### DISCUSSION TITLE
405 Method Not Allowed

### DISCUSSION THREAD

**Discussion by asatwik218** [vercel/community#3520] [Help]
### Summary

I am trying to send base64 image data from a app route to an API route in nextjs 13. Everyting works fine on localhost but on deployment on vercel I get a 405 method not allowed error. I can't find the issue . Please help me out. 

link to repo : [https://github.com/asatwik218/FacePay](url) . 
I am having trouble accessing the POST function on /api/face/index
The get function on /api/face/list works perfectly fine.

### Steps to Reproduce

I dont how to reproduce it locally coz it is not having a problem locally .

**Comment by amyegan** [comment]
Hey there. It looks like your repo is private, so we can't see the code. There are a few other discussions about 405 errors which could be related to your situation:

- https://github.com/vercel/next.js/discussions/51984
- https://github.com/vercel/community/discussions/3081
- https://github.com/vercel/community/discussions/3138

**Reply by asatwik218** [reply]
please check it out it's not private https://github.com/asatwik218/FacePay
